Podfile

```diff
@@ -4,7 +4,10 @@ platform :ios, '9.0'
 
 target :'AsyncDisplayKitTests' do
   pod 'OCMock', '=3.4.1' # 3.4.2 currently has issues.
-  pod 'FBSnapshotTestCase/Core', '~> 2.1'
+
+  # TODO Move to Uber iOSSnapshotTestCase
+  pod 'FBSnapshotTestCase/Core', :git => 'https://github.com/facebookarchive/ios-snapshot-test-case/', :tag => '2.1.4'
```

Comment:
We should plan to upgrade and point to the near future: https://github.com/uber/ios-snapshot-test-case